integrations/acquisition/covid_hosp/state_timeseries/test_scenarios.py

```diff
@@ -19,47 +20,36 @@
     'delphi.epidata.acquisition.covid_hosp.state_timeseries.update'
 
 
-class AcquisitionTests(unittest.TestCase):
+class AcquisitionTests(CovidHospTestCase):
 
   def setUp(self):
     """Perform per-test setup."""
 
     # configure test data
     self.test_utils = UnitTestUtils(__file__)
 
-    # use the local instance of the Epidata API
-    Epidata.BASE_URL = 'http://delphi_web_epidata/epidata/api.php'
-
-    # use the local instance of the epidata database
-    secrets.db.host = 'delphi_database_epidata'
-    secrets.db.epi = ('user', 'pass')
-
     # clear relevant tables
-    with Database.connect() as db:
-      with db.new_cursor() as cur:
-        cur.execute('truncate table covid_hosp_state_daily')
-        cur.execute('truncate table covid_hosp_state_timeseries')
-        cur.execute('truncate table covid_hosp_meta')
+    # TODO: get these names dynamically from the YAML file once all are available
+    super().setUp(Database(), [
+      'covid_hosp_state_daily', # TODO: no need to truncate this after covid_hosp split https://github.com/cmu-delphi/delphi-epidata/pull/1226
+      'covid_hosp_state_timeseries',
+      'covid_hosp_meta'
+    ])
```

Comment:
with my other suggested changes to the base `CovidHospTestCase`, all of this `setUp()` method can be replaced with just:
```
  db_class = Database
  test_util_context = __file__
  # TODO: no need for the following after covid_hosp table split is merged:
  #   (in https://github.com/cmu-delphi/delphi-epidata/pull/1126 )
  extra_tables_used = ['covid_hosp_state_daily']
```
thereby getting `super().setup()` for free